lms/views/api/canvas/authorize.py

```diff
@@ -51,6 +54,9 @@ def authorize(request):
 
     scopes = FILES_SCOPES
 
+    if application_instance.settings.get("canvas", "file_picker_folders"):
+        scopes += FOLDERS_SCOPES
```

Comment:
Do we want to make this setting be enabled by default for new application instances (that have a Canvas developer key) so that we get all the needed scopes from the start?

Reply:
I'd like to first sync with support so the new scope is actually added on the canvas side for new installs.